Apply update scripts to source and parameter files

gassmoeller opened the pull request:
Apply our parameter and source file update scripts to eliminate some outdated variables and input parameters.

tjhei reviewed (approved):
So the scripts apply cleanly? Can we do this in CI as part of the indent step?

gassmoeller commented:
Unfortunately they do not apply cleanly, some of the scripts to update from 1.5.0 to 2.0.0 collide with some comments and names of newer parameter and source files and we also reverted one of the changes of that script (we now again have a postprocessor viscous dissipation statistics that we removed before). The new ones for 2.2.0 apply cleanly though. We could decide to only apply a subset of scripts? But that would have to be maintained for future versions (changing which scripts to apply as part of the CI).

tjhei commented:
> The new ones for 2.2.0 apply cleanly though. We could decide to only apply a subset of scripts? But that would have to be maintained for future versions (changing which scripts to apply as part of the CI).

I think that would be reasonable. We could create a folder "current" with symlinks to the latest update scripts and just run those.
